### ISSUE TITLE
ServiceLocation namespace is not found

### ISSUE THREAD

**Issue by Slashernator** [IG-Group/ig-webapi-dotnet-sample#12]
Hi,
Tried downloading and running the application but no matter how many references or help articles I read, the namespace will not reference the "using Microsoft.Practices.ServiceLocation".
Apparently this namespace is no longer valid.
Any ideas on how to correct?

**Comment by deVaz** [comment]
hi @Slashernator 

Did you manage to solve this ?

regards,

**Comment by Slashernator** [comment]
Hi @deVaz ,

Sorry, I didn't solve it.  Just gave up in the end.

Cheers

**Comment by deVaz** [comment]
Hi @teebeast ,

Thank you so much !!
Quite eager to start coding again ( haven't coded for a while) 
Let me have a look at that beauty

**Comment by deVaz** [comment]
Hi @teebeast, it fails to compile and I think it comes from <Project Sdk="Microsoft.NET.Sdk.WindowsDesktop"> as I'm running the project on Visual studio code on Mac. Should I write anything else instead ?

thank you 

Regards,

Charles

**Comment by teebeast** [comment]
No, there is no chance. You have to rewrite all the ui-stuff for Mac and you have to decouple the logic from the ui. Its better to write a complete new program with a clean API.

**Comment by deVaz** [comment]
Understood, I'm gonna install parrallel desktop I should be able to run the window version
